pymc3/tests/test_distributions.py

```diff
@@ -2511,100 +2508,100 @@ def test_ex_gaussian_cdf_outside_edges(self):
-            lambda value, b, sigma: sp.rice.logpdf(value, b=b, loc=0, scale=sigma),
+        @pytest.mark.xfail(
+            condition=(aesara.config.floatX == "float32"),
+            reason="Some combinations underflow to -inf in float32 in pymc version",
         )
+        def test_rice(self):
+            self.check_logp(
+                Rice,
+                Rplus,
+                {"b": Rplus, "sigma": Rplusbig},
+                lambda value, b, sigma: sp.rice.logpdf(value, b=b, loc=0, scale=sigma),
+            )
 
-    def test_rice_nu(self):
-        self.check_logp(
-            Rice,
-            Rplus,
-            {"nu": Rplus, "sigma": Rplusbig},
-            lambda value, nu, sigma: sp.rice.logpdf(value, b=nu / sigma, loc=0, scale=sigma),
-        )
+        def test_rice_nu(self):
+            self.check_logp(
+                Rice,
+                Rplus,
+                {"nu": Rplus, "sigma": Rplusbig},
+                lambda value, nu, sigma: sp.rice.logpdf(value, b=nu / sigma, loc=0, scale=sigma),
+            )
 
-    def test_moyal_logp(self):
-        # Using a custom domain, because the standard `R` domain undeflows with scipy in float64
-        value_domain = Domain([-inf, -1.5, -1, -0.01, 0.0, 0.01, 1, 1.5, inf])
-        self.check_logp(
-            Moyal,
-            value_domain,
-            {"mu": R, "sigma": Rplusbig},
-            lambda value, mu, sigma: floatX(sp.moyal.logpdf(value, mu, sigma)),
-        )
+        def test_moyal_logp(self):
+            # Using a custom domain, because the standard `R` domain undeflows with scipy in float64
+            value_domain = Domain([-inf, -1.5, -1, -0.01, 0.0, 0.01, 1, 1.5, inf])
+            self.check_logp(
+                Moyal,
+                value_domain,
+                {"mu": R, "sigma": Rplusbig},
+                lambda value, mu, sigma: floatX(sp.moyal.logpdf(value, mu, sigma)),
+            )
 
-    @pytest.mark.xfail(
-        condition=(aesara.config.floatX == "float32"),
-        reason="Pymc3 underflows earlier than scipy on float32",
-    )
-    def test_moyal_logcdf(self):
-        self.check_logcdf(
-            Moyal,
-            R,
-            {"mu": R, "sigma": Rplusbig},
-            lambda value, mu, sigma: floatX(sp.moyal.logcdf(value, mu, sigma)),
+        @pytest.mark.xfail(
+            condition=(aesara.config.floatX == "float32"),
+            reason="Pymc3 underflows earlier than scipy on float32",
         )
+        def test_moyal_logcdf(self):
+            self.check_logcdf(
+                Moyal,
+                R,
+                {"mu": R, "sigma": Rplusbig},
+                lambda value, mu, sigma: floatX(sp.moyal.logcdf(value, mu, sigma)),
+            )
 
     @pytest.mark.xfail(condition=(aesara.config.floatX == "float32"), reason="Fails on float32")
-    @pytest.mark.xfail(reason="Distribution not refactored yet")
+    @pytest.mark.xfail(reason="Test not yet refactored for new version")
```

Comment:
This will need to be refactored for this PR, otherwise we have no test to cover the `logp` method of the `Interpolated`, or do we?

Reply:
Yea, I plan on doing exactly that in this PR. It seems some problem with the arguments not being passed correctly.